datafusion/proto/tests/cases/roundtrip_logical_plan.rs

```diff
@@ -31,8 +31,10 @@ use datafusion::datasource::TableProvider;
 use datafusion::execution::context::SessionState;
 use datafusion::execution::runtime_env::{RuntimeConfig, RuntimeEnv};
 use datafusion::execution::FunctionRegistry;
-use datafusion::functions_aggregate::expr_fn::{covar_pop, covar_samp, first_value};
+use datafusion::functions_aggregate::covariance::{covar_pop, covar_samp};
+use datafusion::functions_aggregate::expr_fn::first_value;
```

Comment:
Can you please change this to `use` the exports from `datafusion::functions_aggregate::expr_fn` (which I think is the intended external facing API)?

Reply:
Updated to use exports from `expr_fn`.